api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -77,51 +69,44 @@ public class AppFrameworkServiceImpl extends BaseOpenmrsService implements AppFr
 
 	private AppFrameworkConfig appFrameworkConfig;
 
-	private ScriptEngine javascriptEngine;
```

Comment:
Why remove this variable?

Reply:
We don't need this with GraalVM.

Reply:
What happens if we leave this variable and continue using it as before? I would like us to start by doing very minimal changes and do the refactoring or improvements later on in another ticket. Does this make sense?